### ISSUE TITLE
Patched classes via --patch-module are ignored

### ISSUE THREAD

**Issue by rsmckinney** [manifold-systems/manifold#209]
**Describe the bug**
Patched classes via --patch-module are ignored. See the example from [stack overflow](https://stackoverflow.com/questions/63420048/how-can-we-overwrite-java-base-java-lang-integer-from-openjdk-11-using-patch-m).

**Comment by rsmckinney** [comment]
Use the example from the stackoverflow post, but add `-Xplugin:Manifold` and classpath for both compile commands e.g.,
```
-Xplugin:Manifold -classpath C:\manifold-systems\manifold\manifold-core-parent\manifold\target\manifold-2020.1.23-SNAPSHOT.jar;C:\manifold-systems\manifold\manifold-util\target\manifold-util-2020.1.23-SNAPSHOT.jar;C:\manifold-systems\manifold\manifold-core-parent\manifold-rt\target\manifold-rt-2020.1.23-SNAPSHOT.jar
```
From the stackoverflow post:
>Here is a sample project that demonstrates how to shadow java.lang.Integer: >https://dl.dropbox.com/s/6swet2k89vukxbm/patch.zip
>
>Compile:
>```
>javac --patch-module java.base=src -d mypatches/java.base src/java.base/java/lang/Integer.java
>javac --patch-module java.base=mypatches/java.base -d target src/main/java/com/example/MyInteger.java >src/main/java/com/example/RunMe.java
>```
>
>Run:
>```
>java --patch-module java.base=mypatches/java.base -classpath target com.example.RunMe
>```

**Comment by rsmckinney** [comment]
Fix released with v2020.1.24
